tck/src/main/java/org/eclipse/microprofile/concurrency/tck/MPConfigTest.java

```diff
@@ -405,6 +411,81 @@ public void overrideContextPropagationForThreadContextParameterWithConfig() {
             Thread.currentThread().setPriority(originalPriority);
         }
     }
+    
+    /**
+     * Verify MicroProfile Config overrides the propagated attribute to enabled
+     * an implied clear of all remaining.
+     * @throws TimeoutException 
+     * @throws ExecutionException 
+     * @throws InterruptedException 
+     */
+    @Test
+    public void overrideContextPropagationForThreadContextWithImpliedCleared() throws InterruptedException, ExecutionException, TimeoutException {
+        ExecutorService unmanagedThreads = null;
+        
+        // Expected config is propagated=Buffer; cleared=Remaining; unchanged=Label
```

Comment:
Based on its absence from MP Config and default value for cleared, I would expect the config to be cleared=Transaction here.  And then, given that none of the config specifies ALL_REMAINING, this implies that all remaining context is cleared.